components/SingleArticle.jsx

```diff
@@ -21,6 +26,11 @@ function SingleArticle() {
       </section>
       <section id="comment-section">
         <h6>Comments:</h6>
+        <ul>
```

Comment:
In case there are no comments yet (like at article 37), let the user know about it with a message rendered in place of the comments.